etc/config/c++.amazon.properties

```diff
@@ -165,6 +165,10 @@ compiler.clang_autonsdmi.exe=/opt/compiler-explorer/clang-autonsdmi-trunk/bin/cl
 compiler.clang_autonsdmi.semver=(experimental auto NSDMI)
 compiler.clang_autonsdmi.options=-std=c++2a -stdlib=libc++
 compiler.clang_autonsdmi.notification=Experimental auto NSDMI; see <a href="https://cor3ntin.github.io/posts/auto_nsdmi/" target="_blank" rel="noopener noreferrer">this blog post <sup><small class="glyphicon glyphicon-new-window opens-new-window" title="Opens in a new window"></small></sup></a> for more information
+compiler.clang_lifetime.exe=/opt/compiler-explorer/clang-lifetime-trunk/bin/clang++
+compiler.clang_lifetime.semver=(experimental -Wlifetime)
+compiler.clang_lifetime.options=-Wlifetime
+compiler.clang_lifetime.notification=Lifetime profile checker based on Herb Sutter&#39;s paper; see <a href="https://herbsutter.com/2018/09/20/lifetime-profile-v1-0-posted/" target="_blank" rel="noopener noreferrer">this blog post <sup><small class="glyphicon glyphicon-new-window opens-new-window" title="Opens in a new window"></small></sup></a> for more information
```

Comment:
Sadly my terrible config file parser will baulk at this: `#` is a comment and so the bit after `Sutter&` will be dropped. Can you use a normal apostrophe here?